# Message to send information about Vehicles in the scene

# id : The idenfification of the vehicle
# pos_x : Vehicle x position
# pos_y : Vehicle y position
# heading : Vehicle heading (Yaw angle)
# velocity : Linear velocity of the vehicle
# max_vel : Maximum velocity

int16   vehicle_id
float32 pos_x
float32 pos_y
float32 heading
float32 velocity
float32 max_vel
